### ISSUE TITLE
Accounts creation on empty request

### ISSUE THREAD

**Issue by thibaultCha** [Mashape/kong#96]
```shell
curl -i -X POST -d "" http://localhost:8001/accounts/
HTTP/1.1 201 Created
Date: Wed, 25 Mar 2015 23:10:15 GMT
Content-Type: application/json
Transfer-Encoding: chunked
Connection: keep-alive
Server: kong/0.1.0beta-1

{"created_at":1427325015000,"id":"39edc72b-a67e-4f34-cf84-92b137b79311"}
```

We should probably have a least one parameter to be able to create an account?

**Comment by thefosk** [comment]
Given the recent updates, a `Consumer` (instead of `Account`) now has the following properties:

* `custom_id`
* `username`

At least one should be sent. Either the `custom_id` (if the user already has a database he wants to map the consumer to), or `username` if he relies on Kong as the primary database, which are the only two possible use-cases.

**Comment by thibaultCha** [comment]
Put that in #100 if we have time

**Comment by thibaultCha** [comment]
Closed by #102. Now an` account` (renamed `consumer` can only be created if given a `username` or a `custom_id`
